data/migrations/20190211232813_add_notes_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("notes", table => {
+    table.increments();
+    table.string("title").notNullable();
+    table.string("textBody", 2000);
```

Comment:
I like that you're including a text limit here. It can save you from a malicious user.